deploy/crds/istio_v1alpha2_istiocontrolplane_cr.yaml

```diff
@@ -2,8 +2,17 @@
 apiVersion: install.istio.io/v1alpha2
 kind: IstioControlPlane
 metadata:
-  namespace: istio-operator
+  namespace: istio-system
```

Comment:
Is this intent? Do we also need to update the namespace in operator chart: https://github.com/istio/operator/tree/master/data/operator

Reply:
Yes, it's intentional. The ICP should be in istio-system and the operator should be watching that. The operator resources should remain in istio-operator though.